internal/dbmodel/sql/postgres/023_add_model_ssh_keys.up.sql

```diff
@@ -0,0 +1,7 @@
+-- add model association to user ssh keys
+ALTER TABLE ssh_keys DROP CONSTRAINT unique_identity_ssh_key;
+
+ALTER TABLE ssh_keys ADD COLUMN model_uuid TEXT,
+    ADD CONSTRAINT fk FOREIGN KEY(model_uuid) REFERENCES models(uuid) ON DELETE CASCADE;
```

Comment:
Could we format this slightly differently 
```suggestion
ALTER TABLE ssh_keys 
    ADD COLUMN model_uuid TEXT,
    ADD CONSTRAINT fk FOREIGN KEY(model_uuid) REFERENCES models(uuid) ON DELETE CASCADE;
```